form.yml

```diff
@@ -60,12 +72,82 @@ cluster:
       $type: boolean
       $default: false
       $optional: true
-    device:
-      $name: SBD device url
-      $type: text
-      $placeholder: /dev/vdb
+    device_list:
+      $visibleIf: .diskless_checkbox == false
+      $name: SBD Device(s)
       $optional: true
-
+      $type: edit-group
+      $itemName: ""
+      $minItems: 0
+      $prototype:
+        $name: SBD device url
+        $optional: true
+        $type: text
+    configure_sbd_checkbox:
+      $name: Configure sbd cluster resource
+      $type: boolean
+      $default: false
+      $help: This updates the resource stonith-sbd created if sbd is enabled. sbd configuration will be overwritten if the resource is defined in the configure template
+    configure_resource:
+      $name: Configure sbd resource
+      $visibleIf: .configure_sbd_checkbox == true
+      $optional: true
+      $type: group
+      force_configure_checkbox:
+        $name: Force commit
+        $type: boolean
+        $default: false
+        $help: Force commit. Use with caution, this can allow invalid parameters in the configuration
+      configure_resource_params:
+        $name: Configure resource params
+        $type: boolean
+        $default: false
+        $help: Mark this option to configure sbd resource parameters
+      resource_params:
+        $name: params
+        $visibleIf: .configure_resource_params == true
+        $type: group
+        pcmk_delay_max:
+          $name: Pacemaker max delay
+          $type: text
+          $default: 15
+          $help: Delay of up to the time specified before executing stonith actions
+      configure_resource_operations:
+        $name: Configure resource operations
+        $type: boolean
+        $default: false
+        $help: Mark this option to configure sbd resource operations like monitor        
+      resource_op:  
+        $name: op
+        $visibleIf: .configure_resource_operations == true
+        $type: group
+        monitor:
+          $name: monitor
+          $type: group
+          monitor_timeout:
+            $name: timout
+            $type: text
+            $default: 15
+            $help: Delay of up to the time specified before executing operation
+          monitor_interval:
+            $name: interval
+            $type: text
+            $default: 15
+            $help: Frequency to perform the operation
+      configure_resource_meta:
+        $name: Configure meta attributes
+        $type: boolean
+        $default: false
+        $help: Mark this option to configure meta attributes
+      resource_meta:
+        $name: meta
+        $visibleIf: .configure_resource_meta == true
+        $type: group
+        resource_stickiness:
```

Comment:
The `resource_stickiness` is only an example. This should be a list of `key/value` entries